assets/keymaps/linux/sublime_text.json

```diff
@@ -4,19 +4,41 @@
       "ctrl-shift-[": "pane::ActivatePrevItem",
       "ctrl-shift-]": "pane::ActivateNextItem",
       "ctrl-pageup": "pane::ActivatePrevItem",
-      "ctrl-pagedown": "pane::ActivateNextItem"
+      "ctrl-pagedown": "pane::ActivateNextItem",
+      "ctrl-1": ["workspace::ActivatePane", 0],
+      "ctrl-2": ["workspace::ActivatePane", 1],
+      "ctrl-3": ["workspace::ActivatePane", 2],
+      "ctrl-4": ["workspace::ActivatePane", 3],
+      "ctrl-5": ["workspace::ActivatePane", 4],
+      "ctrl-6": ["workspace::ActivatePane", 5],
+      "ctrl-7": ["workspace::ActivatePane", 6],
+      "ctrl-8": ["workspace::ActivatePane", 7],
+      "ctrl-9": ["workspace::ActivatePane", 8],
+      "ctrl-shift-1": ["workspace::MoveItemToPane", 0],
+      "ctrl-shift-2": ["workspace::MoveItemToPane", 1],
+      "ctrl-shift-3": ["workspace::MoveItemToPane", 2],
+      "ctrl-shift-4": ["workspace::MoveItemToPane", 3],
+      "ctrl-shift-5": ["workspace::MoveItemToPane", 4],
+      "ctrl-shift-6": ["workspace::MoveItemToPane", 5],
+      "ctrl-shift-7": ["workspace::MoveItemToPane", 6],
+      "ctrl-shift-8": ["workspace::MoveItemToPane", 7],
+      "ctrl-shift-9": ["workspace::MoveItemToPane", 8]
     }
   },
   {
     "context": "Editor",
     "bindings": {
+      "ctrl-alt-up": "editor::AddSelectionAbove",
+      "ctrl-alt-down": "editor::AddSelectionBelow",
       "ctrl-shift-up": "editor::MoveLineUp",
       "ctrl-shift-down": "editor::MoveLineDown",
       "ctrl-shift-m": "editor::SelectLargerSyntaxNode",
       "ctrl-shift-l": "editor::SplitSelectionIntoLines",
       "ctrl-shift-a": "editor::SelectLargerSyntaxNode",
       "ctrl-shift-d": "editor::DuplicateLineDown",
       "alt-f3": "editor::SelectAllMatches", // find_all_under
+      "f9": "editor::SortLinesCaseSensitive",
```

Comment:
These two lines look unrelated?
I see two more below, and same question to `AddSelectionAbove` and `AddSelectionBelow`.

Reply:
Yep. Those are not related to this PR, just missing default ones. Felt like creating a separate PR for this would be an overkill and I can just change a few lines while I'm here. Is this a no-no? Should I remove them?

Reply:
I don't want to be overly pedantic, just the keybindings is somewhat tough topic and I'd prefer to see them separate, so we could ask @\notpeter or someone to check on them and remember that they are added.

F* bindings seem quite safe to add, esp. into sublime-only keymap, so up to you.